tests/test-timber-pagination.php

```diff
@@ -123,7 +123,7 @@ function testDoubleEncodedPaginationUrl() {
 		query_posts('post_type=portfolio&paged=3');
 
 		$link = Timber::compile_string("{{ posts.pagination.next.link|e('esc_url') }}", array(
-			'posts' => new Timber\PostQuery(),
+			'posts' => new PostQuery($GLOBALS['wp_query']),
```

Comment:
I’m just picking one your changes randomly… there are quite a few of those.

Is this an example for the following comment of yours?

> I will continue leaning on skipping tests temporarily for known issues, so the tests can be a kind of todo-list once both PRs are in 2.x-factories. Sound good?

Are usages of `new PostQuery()` something that still needs work or are you deliberately using it here to test specific functionality in `PostQuery`?

Reply:
I tried to go through and replace all these instances in the tests, except for the skipped ones where there are open questions. Generally I think we should test at the highest level possible.